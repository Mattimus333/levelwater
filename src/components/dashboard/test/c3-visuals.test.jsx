import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import c3Visuals from '../c3-visuals.jsx';

describe('c3 visuals', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<c3Visuals />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
