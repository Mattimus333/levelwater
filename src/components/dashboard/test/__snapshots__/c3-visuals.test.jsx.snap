exports[`c3 visuals renders a snapshot 1`] = `<c3Visuals />`;
